### ISSUE TITLE
Check error logging on EmbedsController#show

### ISSUE THREAD

**Issue by juanignaciosl** [CartoDB/cartodb#12471]
### Context

While debugging CartoDB/oncall/issues/45, we detected some 500 errors at unicorn.production.log:

```
Processing by Carto::Builder::Public::EmbedsController#show as HTML
  Parameters: {"user_domain"=>"bbc001", "visualization_id"=>"5b1a1828-67f4-11e7-a5da-xxxxxxxxxxxx"}
  Rendered public/HTML5.html (0.3ms)
Completed 500 Internal Server Error in 1.4ms (Views: 1.0ms | Models: 0.0ms)
```

But we get no error at Rollbar. It looks like we're not tracing there, at least some errors.

**Comment by juanignaciosl** [comment]
This is caused by [this rescue](https://github.com/CartoDB/cartodb/blob/5f7a64d8a2a48504c3765f1eb392de32d77e2149/app/models/carto/layer.rb#L41):

```ruby
[36, 45] in /Users/juanignaciosl/Development/carto/cartodb/app/models/carto/layer.rb
   36:     def tables_from_query(query)
   37:       query.present? ? tables_from_names(affected_table_names(query), user) : []
   38:     rescue => e
   39:       # INFO: this covers changes that CartoDB can't track.
   40:       # For example, if layer SQL contains wrong SQL (uses a table that doesn't exist, or uses an invalid operator).
=> 41:       CartoDB::Logger.debug(message: 'Could not retrieve tables from query', exception: e, user: user, layer: self)
   42:       []
   43:     end
```

Error is logged... as debug :(

**Comment by mayoral** [comment]
Closed via #12475.
